internal/wal_verify_handler_test.go

```diff
@@ -3,30 +3,28 @@ package internal_test
 import (
 	"bytes"
 	"fmt"
-	"github.com/spf13/viper"
 	"github.com/stretchr/testify/assert"
 	"github.com/wal-g/wal-g/internal"
 	"github.com/wal-g/wal-g/utility"
 	"reflect"
 	"testing"
 )
 
-func init() {
-	// Set upload disk concurrency to non-zero value
-	// Please note: this setting affects wal-verify behavior
-	viper.Set(internal.UploadConcurrencySetting, "4")
```

Comment:
why did you remove this line?

Reply:
Because I've added this below:
```
// Please note: this settings affect wal-verify behavior
 // Read more in PostgreSQL.md
 const uploadingSegmentRangeSize = 4
 const delayedSegmentRangeSize = 3
```

Reply:
This was designed to make the logic more straightforward, but it looks like it did the opposite.

Reverted in 6827203